packages/protocol/contracts/stability/Reserve.sol

```diff
@@ -49,6 +49,8 @@ contract Reserve is
 
   uint256 public lastSpendingDay;
   uint256 public spendingLimit;
+  mapping(address => uint256) public erc20SpendingLimit;
+  mapping(address => FixidityLib.Fraction) private tokenDailySpendingRatio;
```

Comment:
New variables go always last as they mess up the storage otherwise, and will also cause test to fail

Reply:
yes yes true. forgot that!